ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ScanConfigClustersTable.tsx

```diff
@@ -111,7 +111,7 @@ function ScanConfigClustersTable({
                     <Tr>
                         <Th sort={getSortParams(0)}>Cluster</Th>
                         <Th sort={getSortParams(1)} width={20}>
-                            Operator status
+                            Scan schedule status
```

Comment:
So it seems this is meant to help distinguish between cluster health and scan configuration issues. Operator status didn't quite convey that. Overall, I think that's fine. I think it would help to get feedback from @mansursyed and Maria as well, just to make sure this is the best approach from their perspective too.

Reply:
I have informed Maria about the changes today during our sync meeting.